tokio/src/runtime/global.rs

```diff
@@ -12,13 +11,12 @@ enum State {
     // default executor not defined
     Empty,
 
+    // Current-thread executor
+    CurrentThread(*const current_thread::Scheduler),
```

Comment:
This module is only enabled with feature `rt-core`, so there is no need to scope this variant.